avl_tree/AvlTree.java

```diff
@@ -0,0 +1,306 @@
+import java.util.Scanner;
+import java.util.NoSuchElementException;
+import java.util.InputMismatchException;
+
+public class AvlTree {
+
+	public static void main(String[] args) {
+		AVL avl = new AVL();
+		Scanner inp = new Scanner(System.in);
+		while (true) {

```

Comment:
I hope you understand the concept of hardcode input @Hsankesara . Please refer other implemented algos/DS to see how we hardcode inputs.